scripts/close_inactive.rb

```diff
@@ -22,11 +22,18 @@ def close_prs(client, repo, config, label, time, message)
     labels = client.labels_for_issue(repo, number).collect{ |x| x[:name] }
     if updated_at < time && labels.include?(label)
       puts "Closing #{number} (#{title}) with latest update #{updated_at}"
-      client.add_comment(repo, number, message % user) if CONFIG[:add_comment]
+      pr_obj = PullRequest.new(config, pr, client)
+
+      final_message = message
+      if pr_obj.dirty?
```

Comment:
Is it really needed to check for dirty? I personally always start with rebasing an outdated PR, even if it's still mergeable.